tests/test_sbi.py

```diff
@@ -95,19 +100,25 @@ def simulator(params):
 
     # choose a random input
     ind = np.random.randint(len(theta))
-
-    nsamples = 20
+    
+    # Check the forward of the embedding network
+    r = embedding_net.forward(torch.FloatTensor(x))
+    unittest.TestCase().assertIsInstance(r, torch.Tensor)
+    unittest.TestCase().assertEqual(r.shape[0], x.shape[0])
+    
+    nsamples = 6
 
     # generate samples from the posterior using accept/reject sampling
     samples = posterior.sample((nsamples,), torch.Tensor(x[ind]).to(device))
-
+    
     # calculate the log_prob for each sample
-    log_prob = posterior.log_prob(samples, torch.Tensor(x[ind]).to(device))
+    # log_prob = posterior.log_prob(samples, torch.Tensor(x[ind]).to(device))
```

Comment:
comment here on purpose?

Reply:
good spot - this shouldn't have been commented out